vscode/src/rbs.ts

```diff
@@ -0,0 +1,60 @@
+import * as vscode from "vscode";
+
+// This class is used to dim RBS signatures in Ruby files
+// For example, these signatures will be dimmed:
+//
+//   #: (String) -> (String | nil)
+//   #: (String) { (String) -> boolish } -> void
+//   #: return: String
+//
+// However, this will not be dimmed:
+//
+//   attr_reader :name #: String
+//
+export class RBS {
```

Comment:
We shouldn't use text editor decorations to change opacity. This stops themes from deciding what opacity they want to apply to tokens and forces everyone to have lower opacity.

As long as we use a specific token for highlighting, themes will be able to decide if they want opacity or not.

Can we please remove this?